stixpy/net/client.py

```diff
@@ -63,6 +98,12 @@ def search(self, *args, **kwargs):
         matchdict = self._get_match_dict(*args, **kwargs)
         levels = matchdict["Level"]
 
+        versions = []
+        for versionAttrType in [Version, VersionU, MinVersion, MinVersionU, MaxVersion, MaxVersionU]:
+            if versionAttrType.__name__ in matchdict:
+                for version in matchdict[versionAttrType.__name__]:
+                    versions.append(versionAttrType(int(version)))
+
```

Comment:
How does this work if I search for multiple products with different min/max versions